irods/test/login_auth_test.py

```diff
@@ -202,6 +221,8 @@ def tearDownClass(cls):
     def setUp(self):
         if not getattr(self, 'envdirs', []):
             self.skipTest('The test_rods_user "{}" does not exist'.format(self.test_rods_user))
+        if os.environ['HOME'] != '/var/lib/irods':
```

Comment:
does this consider non-package-install?    is that important in this scenario?    i think it's probably fine...

Reply:
Ah, need to check if there's a way to get that.  iRODS API?   System package config won't work obvs....

Reply:
i don't think there is - the home directory / what user is running - is dictated by the OS...  let's leave it for now.